src/lib/components/AppSidebar.svelte

```
@@ -0,0 +1,159 @@
+			{
+				title: 'Feedback',
+				url: '#',
+				icon: Send
+			}
+		]
+	};
+</script>
+
+<script lang="ts">
+	import NavMain from '$lib/components/NavMain.svelte';
+	import NavProjects from '$lib/components/NavProjects.svelte';
```

ESLint

[error] 'NavProjects' is defined but never used.

(@typescript-eslint/no-unused-vars)